ui/app/components/auth-saml.hbs

```diff
@@ -4,7 +4,7 @@
 ~}}
 
 {{#if this.canLoginSaml}}
-  <form id="auth-form" onsubmit={{action "startSAMLAuth" @onSubmit (hash role=this.roleName)}}>
+  <form id="auth-form" {{on "submit" (fn this.startSAMLAuth @onSubmit (hash role=@roleName))}}>
```

Comment:
It looks like `@roleName` is an arg and also set by the input below

Reply:
So I think the input below needs to be update?

Reply:
Ah, good catch! I should have done a search for the "action" keyword. This makes a lot more sense.